requirements/edx/base.txt

```diff
@@ -47,7 +47,8 @@ meliae==0.4.0
 mongoengine==0.10.0
 MySQL-python==1.2.5
 networkx==1.7
-nose==1.3.7
+git+https://github.com/edx/nose.git@99c2aff0ff51bf228bfa5482e97e612c97a23245#egg=nose==1.3.7
```

Comment:
@clytwynec we needed to fork nose in order to get past an error in the multiprocess plugin. See https://github.com/nose-devs/nose/pull/955. I'm now thinking perhaps I need to make it v 1.3.7.1 on our fork, and put it in github.txt to ensure it's installed correctly without making things too messy in base.txt.